weave/ops_domain/table.py

```diff
@@ -157,6 +159,10 @@ def _data_is_weave_file_format(data):
 
 def _sample_rows(data: list, max_rows: int = 1000) -> list:
     data_len = len(data)
+
+    if data_len > max_rows and max_rows == 2:
+        return [data[0],  data[-1]]
+
```

Comment:
WIP WIP WIP

This is here because the current `_sample_rows` code does not behave properly when `max_rows` is very low, like `max_rows == 2`. Proper fix incoming.